UPGRADE-3.0.md

```diff
@@ -0,0 +1,10 @@
+UPGRADE FROM 2.x to 3.0
+=======================
+
+### Dependencies
+
+You will need to follow the dependencies upgrade instructions.
+
+## GalleryHasMedia
+
+The `GalleryHasMedia` has been renamed to `GalleryItem`, please adjust your code and be sure to move your assets to the correct database table.
```

Comment:
That it's too long. If you open the file with vim, and type `10Ggqq`, it will be fixed :)